CMakeLists.txt

```diff
@@ -137,6 +137,15 @@ if(TRITON_ONNXRUNTIME_LIB_PATHS STREQUAL "")
   set(TRITON_ONNXRUNTIME_DOCKER_BUILD ON)
 endif()
 
+set(RHEL_BUILD OFF)
+if(LINUX)
+  file(STRINGS "/etc/os-release" DISTRO_ID_LIKE REGEX "ID_LIKE")
+  if(${DISTRO_ID_LIKE} MATCHES "rhel|centos")
+    set(RHEL_BUILD ON)
+  endif(${DISTRO_ID_LIKE} MATCHES "rhel|centos")
+endif(LINUX)
+set(TRITON_CORE_HEADERS_ONLY OFF)
```

Comment:
Why is this needed? I thought this is only needed for building the `core` repository?

Reply:
Totally not needed. This was a copy-paste error.